daemon/logging/logging_outlets.go

```diff
@@ -170,5 +174,121 @@ func (o *SyslogOutlet) WriteEntry(entry logger.Entry) error {
 	default:
 		return o.writer.Err(s) // write as error as reaching this case is in fact an error
 	}
+}
+
+type FileOutlet struct {
+	file      *os.File
+	filename  string
+	formatter EntryFormatter
+	template  *template.Template
+	writer    io.Writer
+}
+
+func (self *FileOutlet) WriteEntry(entry logger.Entry) error {
+	bytes, err := self.formatter.Format(&entry)
+	if err != nil {
+		return err
+	}
+
+	if err := self.reOpenIfNotExists(); err != nil {
+		return nil
```

Comment:
Shouldn't we be returning `err` here too?

Reply:
Thanks, you are right. My typo.